src/coom.jl

```diff
@@ -33,36 +33,41 @@ julia> using TextAnalysis, DataStructures
 ```
 """
 function coo_matrix(::Type{T},
-                    doc::Vector{<:AbstractString},
-                    vocab::OrderedDict{<:AbstractString, Int},
-                    window::Int,
-                    normalize::Bool=true) where T<:AbstractFloat
+    doc::Vector{<:AbstractString},
+    vocab::OrderedDict{<:AbstractString,
+        Int},
+    window::Int,
+    normalize::Bool=true,
+    mode::Symbol=:default) where {T<:AbstractFloat}
```

Comment:
Please describe new argument in the julia-doc above

Reply:
Added the description of the argument mode with an accompanying example.